### DISCUSSION TITLE
Translations show as completed when they are not.

### DISCUSSION THREAD

**Discussion by chimosky** [WeblateOrg/weblate#10673] [Q&A] [answered]
The percentage of translated strings for some components show that most of the strings have been translated but the ratio of the translated and untranslated strings show that's false.

![Screenshot from 2023-12-21 17-32-01](https://github.com/WeblateOrg/weblate/assets/16203834/59747e68-3a31-4c9a-9b1d-642559114702)

**Comment by nijel** [comment, marked as answer]
5.3 release didn't update these stats, 5.3.1 fixed that. These will eventually update. See https://github.com/WeblateOrg/weblate/issues/10508#issuecomment-1862791877

**Reply by chimosky** [reply]
Thank you!

**Reply by chimosky** [reply]
Also noticed that I'd deleted a translation project and got a "Selected to be deleted" prompt if I remember correctly but still see the project and its components, how long would it take for it to be deleted. 

I did a nasty workaround cos I needed the space at the time, deleted the project directly from the docker container.
